app/views/hyrax/my/_facet_limit.html.erb

```diff
@@ -1,4 +1,4 @@
-  <ul class="dropdown-menu facet-values" aria-labelledby="<%= facet_field.field.parameterize %>">
+  <ul class="<%= 'dropdown-menu ' if request.format.html? %>facet-values" aria-labelledby="<%= facet_field.field.parameterize %>">
```

Comment:
How about testing for `action_name` instead? I think that would be a lot clearer as to the intention.